practitioner-service/src/main/java/uk/ac/ox/ndph/mts/practitioner_service/repository/AzureFhirRepository.java

```diff
@@ -0,0 +1,84 @@
+package uk.ac.ox.ndph.mts.practitioner_service.repository;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.hl7.fhir.instance.model.api.IBaseResource;
+import org.hl7.fhir.r4.model.Practitioner;
+import org.hl7.fhir.r4.model.Bundle;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.hl7.fhir.r4.model.Resource;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import ca.uhn.fhir.context.FhirContext;
+import ca.uhn.fhir.rest.server.exceptions.BaseServerResponseException;
+import ca.uhn.fhir.util.BundleUtil;
+import uk.ac.ox.ndph.mts.practitioner_service.exception.RestException;
+
+/**
+ * Implement FhirRepository interface using HAPI client sdk and backed up by an Azure
```

Comment:
can you explain why Hapi client? or where do I see that is Hapi

also isnt the name should be 'backed up by an Azure FHIR server'? (instead of store?)